docker/Dockerfile.checks-agent

```diff
@@ -0,0 +1,76 @@
+ARG BUILD_IMAGE
```

Comment:
Not something to address in this PR, but mostly so I don't forget: it strikes me that we should collapse this and `Dockerfile.agent-data-plane` into a single Dockerfile that takes an argument that we pass to `cargo build` for which binary to build, since everything else is identical.

Reply:
I can take care of that, once we decide to merge this to main